resources/frontend_client/app/instance-loading-page/styles.css

```diff
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+  color: #4C5773;
+  font-family: Lato, 'Helvetica Neue', Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  height: 100%;
+  min-height: 100%;
+  position: relative;
+}
+
+body {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+#status {
+  margin-top: 125px;
+  font-size: 14px;
+  color: #949AAB;
+  width: 400px;
+}
+
+progress {
+  margin-top: 25px;
+}
+
+progress[value] {
+  -webkit-appearance: none;
+  appearance: none;
+  border: 1px solid #509EE3;
+  border-radius: 8px;
+  height: 8px;
+  width: 400px;
+}
+
+progress[value]::-webkit-progress-bar {
+  background-color: white;
+  border-radius: 8px;
+}
+
+progress[value]::-webkit-progress-value {
+  background-color: green;
+  background-color: #4A90E2;
+  border-radius: 8px;
+}
+
+
+.heading {
+  box-shadow: 0px 5px 6px rgba(80, 158, 227, 0.16), 0px 12px 24px rgba(148, 154, 171, 0.08);
```

Comment:
Back in the day one of the best practices in CSS was not to use units after zero. We're not adhering to this principle anywhere else in the code base, so consider this as a nitpicking and a nice to.

Maybe we should apply that everywhere?

Reply:
Yup, this was a copy + paste from Figma, and this code is not linted for config reasons.

Will improve manually.